src/app/component/core/search-popup/search-popup.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-template [ngIf]="isSearchClicked">
+  <div class="search-bar-wrapper">
+    <div class="search-icon"></div>
+      <input class = "search-field" type="text" placeholder="Search">
```

Comment:
Please, write attributes in separate rows

Reply:
fixed